web/src/settings_components.ts

```diff
@@ -830,6 +830,16 @@ export function save_discard_widget_status_handler(
     }
 }
 
+export function hide_stream_privacy_warning_compose_banner(): void {
+    const $stream_permissions_warning_banner = $(
+        "#stream_permission_settings .stream-permissions-warning-banner",
+    );
+
+    if ($stream_permissions_warning_banner.length > 0) {
```

Comment:
This will always return true when the stream settings is open and even when the banner is not shown. Instead we can modify this check whether there is banner shown or not inside this container.